.github/workflows/publish-java-cdk-command.yml

```diff
@@ -41,7 +48,7 @@ concurrency:
 env:
   # Use the provided GITREF or default to the branch triggering the workflow.
   GITREF: ${{ github.event.inputs.gitref || github.ref }}
-  FORCE: "${{ github.event.inputs.force == null && 'false' || github.event.inputs.force }}"
+  FORCE: "${{ github.event_name == 'push' || github.event.inputs.force == null && 'false' ||  github.event.inputs.force }}"
```

Comment:
This clause looks dodgy. I realize it was carried over but `|| github.event.inputs.force == null && 'false' ||` doesn't make a ton of sense.

Reply:
is is saying 
if `event_name == push`, return `true`
otherwise, check if `force == null`, in which case you return `false`, otherwise you keep going